html_reference.html

```diff
@@ -0,0 +1,80 @@
+				<div class="socialMediaButtons">
+					<a><i class="fab fa-linkedin"></i></a>
+					<a><i class="fab fab fa-github-square"></i></a>
+					<a><i class="fas fa-share-alt-square"></i></a>
+				</div>
+			</div>
+			<h1 class="myName">milo_girardet  <i class="fas fa-user-astronaut" id="astronaut"></i></h1>					
+		</div>
+	</header>
+		<div class="paragraphs">
+			<div class="aboutMePosition">
+				<h2 class="container">full_stack_developer  <i class="fas fa-code" id="code"></i></h2>
+				<div class="aboutMeContent">
+					<div class="aboutMeText">
+						Palmyra is an ancient city in present-day Homs Governorate, Syria. Inhabited since the Neolithic period, it entered recorded history in the early second millennium 						BC. Palmyra became part of the Roman Empire in the first century AD. Palmyrene merchants established colonies along the Silk Road and the city grew wealthy from 						trade caravans. Many monumental projects were erected, such as the Great Colonnade, the Temple of Bel, and distinctive tower tombs. Palmyra reached the apex of its 						power in the 260s, when its king Odaenathus defeated the Persian emperor Shapur I.
+					</div>
+				</div>
+			</div>
+			<div class="aboutMePosition">
+				<h2 class="container">employment_history  <i class="fas fa-briefcase" id="briefcase"></i></h2>
+				<div class="aboutMeContent">
+					<div class="aboutMeText">
+						Palmyra is an ancient city in present-day Homs Governorate, Syria. Inhabited since the Neolithic period, it entered recorded history in the early second millennium 						BC. Palmyra became part of the Roman Empire in the first century AD. Palmyrene merchants established colonies along the Silk Road and the city grew wealthy from 						trade caravans. Many monumental projects were erected, such as the Great Colonnade, the Temple of Bel, and distinctive tower tombs. Palmyra reached the apex of its 						power in the 260s, when its king Odaenathus defeated the Persian emperor Shapur I.
+					</div>
+				</div>
+			</div>
+			<div class="aboutMePosition">
+				<h2 class="container">extra_curricular <i class="fas fa-biking" id="biking"></i></h2>
+				<div class="aboutMeContent">
+					<div class="aboutMeText">
+						Palmyra is an ancient city in present-day Homs Governorate, Syria. Inhabited since the Neolithic period, it entered recorded history in the early second millennium 						BC. Palmyra became part of the Roman Empire in the first century AD. Palmyrene merchants established colonies along the Silk Road and the city grew wealthy from 						trade caravans. Many monumental projects were erected, such as the Great Colonnade, the Temple of Bel, and distinctive tower tombs. Palmyra reached the apex of its 						power in the 260s, when its king Odaenathus defeated the Persian emperor Shapur I.
+					</div>
+				</div>
+			</div>
+		</div>
+		<h3 class="container">portfolio  <i class="fas fa-palette" id="palette"></i></h3>
```

Comment:
Does this h3 need the container class? Seems to work without

Reply:
pushes portfolio inline with paragraphs.